theme_zap/views/snippets/s_numbers.xml

```diff
@@ -2,10 +2,6 @@
 <odoo>
 
 <template id="s_numbers" inherit_id="website.s_numbers">
```

Comment:
@chgo-odoo , something is off here:
![image](https://github.com/user-attachments/assets/263b7d1f-3396-43ad-95a1-3656ed9c2ec8)
Could you investigate? Eventually you should open a follow-up task, thanks!

Reply:
yeah I'm aware of some issue in some themes, I've already created two follow-up task, there is also an issue in text-cover, will move them forward asap 👍